notebooks/Inference_from_SavedModel.ipynb

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
<p>Let's serialize a couple of images from the validation split in our repository to avoid installing <code>datasets</code> and getting the test images this way.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/semantic-segmentation-ml-pipeline/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>